### DISCUSSION TITLE
Deploying and Endpoint for Huggingface Model

### DISCUSSION THREAD

**Discussion by cenkbircanoglu** [aws/sagemaker-python-sdk#2455] [Help]
I trained a Huggingface model by using HuggingfaceEstimator class. when I tried to deploy this estimator, I saw that `create_model` function is not implemented for this estimator class. Yes, I know I can deploy the model with some changes as Pytorch/Tensorflow. 

Is this interface going to implemented soon or not?

**Comment by ahsan-z-khan** [comment]
Hi @cenkbircanoglu ,

Thank you for using SageMaker.

We will update soon on the `create_model` function.

**Comment by icywang86rui** [comment]
https://github.com/aws/sagemaker-python-sdk/pull/2308
